app/models/inning.rb

```diff
@@ -23,6 +23,39 @@ class Inning < ActiveRecord::Base
   # TODO: Switch to after_commit and use https://github.com/grosser/test_after_commit
   after_save :update_game_score, unless: :destroyed?
 
+  attr_accessor :man_on_first, :man_on_second, :man_on_third
+
+  def play!
+    while !completed? do
+      play_next_at_bat! 
+      save!
+      break if game.going_final?
+    end
+  end
+
+  def play_next_at_bat!
+    hitter_index = game.hitting_team == "away" ? game.increment_away_hitter_index : game.increment_home_hitter_index
+    hitter = lineup[hitter_index]
+
+    plate_appearance = plate_appearances.create!(
+      pitcher: pitcher,
+      batter: lineup[hitter_index],
+      runner_on_first: man_on_first,
+      runner_on_second: man_on_second,
+      runner_on_third: man_on_third
+    )
+
+    puts "#{hitter.first_name} #{hitter.last_name} #{plate_appearance.outcome.code}"
```

Comment:
You may want to put a conditional here so that you don't see print statements when running RSpec